rpm/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# This script will package the `update` script into a `rpm` package.
+
+declare -r NAME="update"
+declare -r BUILD_DIR="$(pwd)"
+declare -r VERSION="$(grep "version=" "${BUILD_DIR}/${NAME}" | cut -d"\"" -f2 | cut -d"v" -f2)"
+
```

Comment:
```suggestion
if command -v dnf &>/dev/null; then
declare -r PKGMANAGER="dnf"
elif command -v yum &>/dev/null; then
declare -r PKGMANAGER="yum"
else
echo "No suitable package manager found!! Exiting..."; exit 1
fi

```

I lust tested on CentOS which doesn't have `dnf`, so the script failed. This will add support for CentOS